<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1977</width>
    <height>1094</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Test</string>
         </attribute>
         <widget class="QTextEdit" name="serialportresulttxt">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>750</y>
            <width>1001</width>
            <height>221</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>1011</width>
            <height>221</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="videocapturelabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>-10</y>
            <width>1001</width>
            <height>751</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="resulttableWidget">
          <property name="geometry">
           <rect>
            <x>1030</x>
            <y>10</y>
            <width>721</width>
            <height>771</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="controlsLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Connection Type</string>
        </property>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>191</width>
           <height>391</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_13">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QTextEdit" name="textEdit_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>580</y>
             <width>1001</width>
             <height>191</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>1011</width>
             <height>221</height>
            </size>
           </property>
          </widget>
          <widget class="QGroupBox" name="PortControlsBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>194</width>
             <height>391</height>
            </rect>
           </property>
           <property name="title">
            <string>PORT CONTROLS</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="resfreshbtn">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="labelPort">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>PORT</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboPort">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>69</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="labelBaud">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>BAUD</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBaud">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>69</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="labelData">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>DATA</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboData">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>69</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="labelParity">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>PARITY</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboParity">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>69</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="labelStop">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>STOP</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboStop">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>69</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gridGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>TEXT CONTROLS</string>
        </property>
        <layout class="QGridLayout" name="TextControlsBox">
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTextEdit" name="textEdit_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Name&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="namelineedit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QTextEdit" name="textEdit_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Age&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="AgeLneEDit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="SaveCheckBox">
             <property name="text">
              <string>SaveData</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_TextEditHide">
             <property name="text">
              <string>Hide TextBox</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_ShowallData">
             <property name="text">
              <string>Show All Incoming Data</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SaveCsv">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionPause_Plot"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionHow_to_use"/>
   <addaction name="separator"/>
   <addaction name="actionRecord_stream"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/serial_port_plotter/play_nor.png</normaloff>
     <normalon>:/serial_port_plotter/play_nor.png</normalon>
     <disabledoff>:/serial_port_plotter/play_dis.png</disabledoff>
     <disabledon>:/serial_port_plotter/play_dis.png</disabledon>
     <activeoff>:/serial_port_plotter/play_act.png</activeoff>
     <activeon>:/serial_port_plotter/play_act.png</activeon>
     <selectedoff>:/serial_port_plotter/play_act.png</selectedoff>
     <selectedon>:/serial_port_plotter/play_act.png</selectedon>:/serial_port_plotter/play_nor.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to COM port</string>
   </property>
  </action>
  <action name="actionPause_Plot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/serial_port_plotter/pause_nor.png</normaloff>
     <normalon>:/serial_port_plotter/pause_nor.png</normalon>
     <disabledoff>:/serial_port_plotter/pause_dis.png</disabledoff>
     <disabledon>:/serial_port_plotter/pause_dis.png</disabledon>
     <activeoff>:/serial_port_plotter/pause_act.png</activeoff>
     <activeon>:/serial_port_plotter/pause_act.png</activeon>
     <selectedoff>:/serial_port_plotter/pause_act.png</selectedoff>
     <selectedon>:/serial_port_plotter/pause_act.png</selectedon>:/serial_port_plotter/pause_nor.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause plot</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/serial_port_plotter/stop_nor.png</normaloff>
     <normalon>:/serial_port_plotter/stop_nor.png</normalon>
     <disabledoff>:/serial_port_plotter/stop_dis.png</disabledoff>
     <disabledon>:/serial_port_plotter/stop_dis.png</disabledon>
     <activeoff>:/serial_port_plotter/stop_act.png</activeoff>
     <activeon>:/serial_port_plotter/stop_act.png</activeon>
     <selectedoff>:/serial_port_plotter/stop_act.png</selectedoff>
     <selectedon>:/serial_port_plotter/stop_act.png</selectedon>:/serial_port_plotter/stop_nor.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect COM port</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/serial_port_plotter/clear_nor.png</normaloff>
     <normalon>:/serial_port_plotter/clear_nor.png</normalon>
     <disabledoff>:/serial_port_plotter/clear_dis.png</disabledoff>
     <disabledon>:/serial_port_plotter/clear_dis.png</disabledon>
     <activeoff>:/serial_port_plotter/clear_act.png</activeoff>
     <activeon>:/serial_port_plotter/clear_act.png</activeon>
     <selectedoff>:/serial_port_plotter/clear_act.png</selectedoff>
     <selectedon>:/serial_port_plotter/clear_act.png</selectedon>:/serial_port_plotter/clear_nor.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear plot data</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="icon">
    <iconset>
     <normaloff>:/serial_port_plotter/help_nor.png</normaloff>
     <normalon>:/serial_port_plotter/help_nor.png</normalon>
     <disabledoff>:/serial_port_plotter/help_dis.png</disabledoff>
     <disabledon>:/serial_port_plotter/help_dis.png</disabledon>
     <activeoff>:/serial_port_plotter/help_act.png</activeoff>
     <activeon>:/serial_port_plotter/help_act.png</activeon>
     <selectedoff>:/serial_port_plotter/help_act.png</selectedoff>
     <selectedon>:/serial_port_plotter/help_act.png</selectedon>:/serial_port_plotter/help_nor.png</iconset>
   </property>
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionRecord_stream">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/line_icon_set/document.png</normaloff>
     <normalon>:/icons/line_icon_set_text/document.png</normalon>
     <disabledoff>:/icons/line_icon_set/document.png</disabledoff>:/icons/line_icon_set/document.png</iconset>
   </property>
   <property name="text">
    <string>Record stream</string>
   </property>
   <property name="toolTip">
    <string>Record the incoming data to a .csv file ( shortcut : s )</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
